contracts/refs/OracleRef.sol

```diff
@@ -141,10 +142,6 @@ abstract contract OracleRef is IOracleRef, CoreRef {
         bool oldDoInvert = doInvert;
         doInvert = newDoInvert;
 
-        if (oldDoInvert != newDoInvert) {
```

Comment:
Why remove this? Was the previous logic wrong?

Reply:
We don't want side effects in this function.